### ISSUE TITLE
权重文件

### ISSUE THREAD

**Issue by Anora0214** [bubbliiiing/classification-keras#3]
您好，想问一下权重文件是来源于哪，针对什么数据集的？因为我想使用一些比较好的权重

**Comment by Anora0214** [comment]
以及百度网盘里的权重文件能否说明一下？比如mobilenet_2_5_224_tf_no_top.h5，这里的2，5是什么意思？非常感谢！

**Comment by Anora0214** [comment]
以及百度网盘里的权重文件能否说明一下？比如mobilenet_2_5_224_tf_no_top.h5，这里的2，5是什么意思？非常感谢！

**Comment by bubbliiiing** [comment]
缩放为0.25呗，alpha=0.25

**Comment by Anora0214** [comment]
大神，你在使用keras做迁移学习的时候，有遇到val_acc最高维持在0.5，而acc有0.9嘛？我也翻找了一些资料，有可能是Keras 中实现 Batch Normalization 层的方式引起（ [https://github.com/keras-team/keras/issues/9214#issuecomment-397916155](url) ），但不知道咋解决。

**Comment by bubbliiiing** [comment]
解冻后应该没问题
